### DISCUSSION TITLE
Can't hear myself or others on a server

### DISCUSSION THREAD

**Discussion by thomas904** [jamulussoftware/jamulus#1652] [Technical Support & Help]
Hi everyone, I'm a jamulus newbie. I have a Behringer UMC404HD audio interface, which is connected to my laptop (Windows 8), and I have installed the Behringer UMC driver. Now I'm able to connect to a jamulus Server, and obviously I can send sound to the server, because the Jamulus mixer LED's react when I play on my keyboard. But I can't hear anything in my headphones! Nor can I hear others playing, singing or speaking, even though I see their mixer LED's moving. Any idea how to fix this?

**Comment by dcorson-ticino-com** [comment]
Hi Thomas,
As you say the leds on the mixboard react then the sound is going out to the server and back to you but not getting from Jamulus to your headphones.  I assume you have them plugged in to the Behringer UMC and the UMC selected as the sound Device in Jamulus' settings.  Windows sound routing is complicated and opaque and I no longer have a Win8 compi to try out.  But there is probably somewhere in your Windows sound settings something sending the sound to another output and not to the Behringer.

**Reply by thomas904** [reply]
thanks dcorson for your reply. Yes, I've selected the UMC as the sound device, and the Winows settings seem to be okay. After rebooting my laptop several times, I can now hear something on server, but the problem is: Either I hear myself but not the others, or the other way round: I hear the others but not myself. Mysterious... What could that be?

**Reply by dcorson-ticino-com** [reply]
You have direct monitoring turned on, I suspect.
Turn direct monitoring off, you need to hear only that which comes back from the server (which includes you) and never the direct signal from your mic/instrument.

**Reply by thomas904** [reply]
Yes, at first I had turned direct monitoring on, which was why I couldn't hear anything on the server. Then I turned it off, so I could hear myself, but not the others, because there was another issue I've detected right now: When entering a server with others playing, my "solo" button is automatically turned on, which prevents me from hearing them. When I turn it off, I can hear them! So the problem is solved
